output/output_functions.py

```diff
@@ -0,0 +1,2 @@
+# This class will be used for storing the functions requested from the issues tab on main branch.

```

Comment:
This sentence could potentially be reworded for clarity. For example, instead of "the functions requested from the issues tab on the branch", you could specify that it will store functions requested from the Github Issue Tracker